src/kaboom.ts

```diff
@@ -7348,6 +7474,9 @@ export default (gopt: KaboomOpt = {}): KaboomCtx => {
 		isVirtualButtonPressed,
 		isVirtualButtonDown,
 		isVirtualButtonReleased,
+		isGamepadButtonPressed,
+		isGamepadButtonDown,
+		isGamepadButtonReleased,
```

Comment:
I think there can also be a function to get the stick value, maybe like `getGamepadStick()`?

Reply:
Sure, will be added